scripts/verdaccioPkgSize.js

```diff
@@ -11,14 +11,21 @@ function verdaccioPkgSize() {
   let verdaccioStorePath = exec('echo ~/.config/verdaccio/storage/').toString().trim();
 
   if (!fs.existsSync(verdaccioStorePath)) {
-    verdaccioStorePath = path.join(__dirname, '..', 'verdaccio', 'storage/');
```

Comment:
unsure why i needed this change, @LFDanLu do you have any insight?

Reply:
Interesting, I assume this was failing on build? Not sure why that would have changed, but maybe the directory structure is different now?